templates/displayApplicants.phtml

```diff
@@ -23,14 +23,14 @@
                     <th class="col-xs-4">Email</th>
                     <th class="col-xs-2">Date of application
                         <div>
-                            <form class="arrowForm" method="get" action="/sort/dateAsc"><button class="arrowBtn" type="submit"><i style="color:#d64405;" class="fas fa-arrow-up"></i></button></form>
-                            <form class="arrowForm" method="get" action="/sort/dateDesc"><button class="arrowBtn" type="submit"><i class="fas fa-arrow-down"></i></button></form>
+                            <form class="arrowForm" method="get"><button name="sort" value="dateAsc" class="arrowBtn" type="submit"><i style="color:#d64405;" class="fas fa-arrow-up"></i></button></form>
```

Comment:
Move inline styling to the appropriate CSS file, the buttons also need to change colour to display which sort method is applied.

Reply:
Removed inline styling. Changed colour of arrows on click.